examples/3d/elixir_euler_ec.jl

```diff
@@ -0,0 +1,58 @@
+
+using OrdinaryDiffEq
+using Trixi
+
+###############################################################################
+# semidiscretization of the compressible euler equations
+
+equations = CompressibleEulerEquations3D(1.4)
+
+initial_condition = Trixi.initial_condition_weak_blast_wave
+
+surface_flux = flux_ranocha
+volume_flux  = flux_ranocha
+solver = DGSEM(3, surface_flux, VolumeIntegralFluxDifferencing(volume_flux))
+
+coordinates_min = (-2, -2, -2)
+coordinates_max = ( 2,  2,  2)
+mesh = TreeMesh(coordinates_min, coordinates_max,
+                initial_refinement_level=3,
+                n_cells_max=100_000)
+
+
+semi = SemidiscretizationHyperbolic(mesh, equations, initial_condition, solver)
+
+
+###############################################################################
+# ODE solvers, callbacks etc.
+
+tspan = (0.0, 0.4)
+ode = semidiscretize(semi, tspan)
+
+summary_callback = SummaryCallback()
+
+stepsize_callback = StepsizeCallback(cfl=0.5)
+
+save_solution = SaveSolutionCallback(interval=100,
+                                     save_initial_solution=true,
+                                     save_final_solution=true,
+                                     solution_variables=:primitive)
+
+save_restart = SaveRestartCallback(interval=10,
```

Comment:
I think this interval is too small (please also check the other tests) - we don't need a restart file every 10 steps (even if it was this way in the TOML file).